widget_tweaks/templatetags/widget_tweaks.py

```diff
@@ -200,3 +200,13 @@ def render(self, context):
             bounded_field = \
                 append_attr(bounded_field, '%s:%s' % (k, v.resolve(context)))
         return str(bounded_field)
+
+
+# ======================== remove_attr tag ==============================
```

Comment:
Is this comment really necessary?

Reply:
I put in there to make it stylistically consistent, since right above it is render_field section. Any future features related to removing attributes/classes can show up in this section.